OracleTuxedo/core/dockerfiles/fix_locations.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# This script modifies the installer scripts and response files to match the local environment.
```

Comment:
Instead of changing paths using an external script, can't you do this the standard Docker way using either `--build-arg` for build time changes or `-e` for runtime changes? That would be much better in terms of 3rd-party orchestration support.